### PR TITLE
added sys import

### PR DESCRIPTION
**Pull request by Nightfurex**

### CODE REVIEW

**Review by DrTodd13 — CHANGES_REQUESTED** [review]
See my comment about removing the line that uses sys.

**Comment by Nightfurex** [comment]
Ooh sorry my bad i misunderstood .

File: numba/parfors/parfor_lowering.py [hunk 1]
```diff
@@ -323,8 +322,7 @@ def _lower_parfor_parallel(lowerer, parfor):
         index_var_typ,
         parfor.races,
         exp_name_to_tuple_var)
-    if config.DEBUG_ARRAY_OPT:
-        sys.stdout.flush()
+
```
Comment (DrTodd13) [inline numba/parfors/parfor_lowering.py:325]: Should only be one blank line here.

**Review by DrTodd13 — CHANGES_REQUESTED** [review]

**Comment by DrTodd13** [comment]
Thanks for making this change.  Just that one additional picky thing about the extra line and I think we're done.

**Review by DrTodd13 — APPROVED** [review]
Thanks!

**Comment by stuartarchibald** [comment]
Thanks for the patch @Nightfurex, thanks for reviewing @DrTodd13.

**Pull request merged**